scripts/haproxy/s3haproxyconfig/s3haproxyconfig/s3_haproxy_config.py

```diff
@@ -57,35 +61,42 @@ def run():
     #Add backend information to haproxy.cfg file
     cfg_file = '/etc/haproxy/haproxy.cfg'
     target = open(cfg_file, "a+")
-    target.write('''#-------S3 Haproxy configuration start--------------------------------
-#---------------------------------------------------------------------
+
+    n = 1
+    env = s3conf_store.get_config("cluster>env_type")
+
+    if env == 'VM':
+        n = 4
+    elif env == 'HW':
+        n = 11
+
+    s3inport = 28081
+    s3auport = 28050
+
+    str1 = '''
+#----------------------------------------------------------------------
 # FrontEnd S3 Configuration
-#---------------------------------------------------------------------
-frontend main
+#----------------------------------------------------------------------
+frontend s3-main
     # s3 server port
     bind 0.0.0.0:80
     #bind 0.0.0.0:443 ssl crt /etc/ssl/stx/stx.pem
 
     option forwardfor
     default_backend app-main
```

Comment:
Change app-main to s3-main as is the backend

Reply:
Done